dist/pythonlibs/testrunner/__init__.py

```diff
@@ -21,10 +21,15 @@
 # default value (10)
 TIMEOUT = int(os.environ.get('RIOT_TEST_TIMEOUT') or 10)
 
+# Some boards can't be resetted during test since this resets the serial
+# connection. Default to 1 for those boards.
+MAKE_RESET_WITH_TERM = int(os.environ.get('MAKE_RESET_WITH_TERM', 1))
```

Comment:
How about we turn this into a variable holding the whole command?
`TESTRUNNER_RESET_CMD = "make reset"`

Reply:
I pushed changes adding `make reset` before `clean term`. I changed the variable name to `TESTRUNNER_RESET_SYNC` I think the expected behavior is more explicit with this name

I can still change it to `TESTRUNNER_RESET_SYNC_CMD` if you prefer.